cypress/support/pages/teams/pageTeams.js

```diff
@@ -0,0 +1,135 @@
+'use strict'
+
+class Teams {
+
+  static #addNewTeamButton ='[data-testid="add-team-btn"]'
+  static #teamName = '[data-testid="team_name"]'
+  static #teamDescription = '[data-testid="description_team"]'
+  static #teamChatActivateCheckBox = '[id="activateRC"]'
```

Comment:
Here we already have `data-testId`. please use it

Reply:
added the data-testid, thanks!